docs/admin/scale/index.md

```diff
@@ -0,0 +1,36 @@
+We regularly scale-test Coder against various reference architectures. Additionally, we provide a [scale testing utility](#scaletest-utility) which can be used in your own environment to give insight on how Coder scales with your deployment's specific templates, images, etc.
+
+## Reference Architectures
+
+| Environment                                       | Users         | Last tested  | Status   |
+| ------------------------------------------------- | ------------- | ------------ | -------- |
+| [Google Kubernetes Engine (GKE)](./gke.md)        | 50, 100, 1000 | Nov 29, 2022 | Complete |
+| [AWS Elastic Kubernetes Service (EKS)](./eks.md)  | 50, 100, 1000 | Nov 29, 2022 | Complete |
+| [Google Compute Engine + Docker](./gce-docker.md) | 15, 50        | Nov 29, 2022 | Complete |
+| [Google Compute Engine + VMs](./gce-vms.md)       | 1000          | Nov 29, 2022 | Complete |
+
+## Scale testing utility
+
+Since Coder's performance is highly dependent on the templates and workflows you support, we recommend using our scale testing utility against your own environments.
+
+For example, this command will do the following:
+
+- create 100 workspaces
+- establish a SSH connection to each workspace
+- run `sleep 3 && echo hello` on each workspace via the web terminal
+- close connections, attempt to delete all workspaces
+- return results (e.g. `99 succeeded, 1 failed to connect`)
+
+```sh
+coder scaletest create-workspaces \
+    --count 100 \
```

Comment:
Side quest: it might be valuable to perform the push-the-limits test. I mean, we're repeating load tests with an increasing number of workspaces until the Coder performance is degraded. Then, we know the exact number of workspaces we can support :)

Reply:
Totally agree - moving this to #5242